Server/Quality_of_Life_changer.Contracts/Commands/AddUserCalendarCommand.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Quality_of_Life_changer.Contracts.Commands;
+
+using MediatR;
+
+public record AddUserCalendarCommand(
+    string Url,
+    string OwnerId,
+    string? CalendarName
+) : IRequest<AddUserCalendarResponse>;
+
+public record AddUserCalendarResponse(string Id, string? CalendarName, string Owner, string Url);
```

Comment:
why do you return calendar data with a response?
in such case CQS and handlers make no sense.
command should not return any data, only modify it

Reply:
it can't return anything, even just id?